assignments/lambda-classes.js

```diff
@@ -1 +1,138 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attrib) {
+        this.name = attrib.name;
+        this.age = attrib.age;
+        this.location = attrib.location;
+        this.gender = attrib.location;
+        }
+        speak() {
+            return `Hello my name is ${this.name}, I am from ${this.location}.`;
+        }
+    }
+
+
+class Instructor extends Person {
+    constructor(instructorAttrib) {
```

Comment:
```suggestion
    constructor(props) {
```
Consider just always using "props" as the variable for your constructor/super. If you think about it, it makes sense, because you're always extending the "prop"erties of the parent class. It will also get you prepared for patterns we use in React.